solutions/src/main/java/ru/sberbank/school/task07/Test.java

```diff
@@ -0,0 +1,38 @@
+package ru.sberbank.school.task07;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Test {
+
+    static String path = "D:\\dev\\java\\SberBank\\homework\\solutions\\src\\main\\java\\ru\\sberbank\\school\\task07\\text.txt";
```

Comment:
Line is longer than 120 characters (found 129).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a872321245>*